DigitalLearningSolutions.Web/ViewModels/Common/SearchablePage/BaseSearchablePageViewModel.cs

```diff
@@ -16,19 +16,23 @@ public abstract class BaseSearchablePageViewModel : BasePaginatedViewModel
 
         public readonly string? SearchString;
 
+        public readonly string? SearchLabel;
+
         protected BaseSearchablePageViewModel(

```

Comment:
I can remember discussing it at some point, but do you remember why we don't have SearchDetails?, FilterDetails? and SortDetails? on this model - If any of these is not specified, it would be equivalent to no-search/no-sort/no-filter

Reply:
We're not going to do this at the moment, it's a large, potentially fairly messy change that will affect other tickets too.